Automattic/wp-calypso
# Earn: Adding Donations

** DO NOT MERGE -- the Donations tool is not yet launched **

This PR adds a Donations card to the Earn page. Original request for this is at pbMlHh-yL-p2.

#### Testing instructions

* Build this branch locally and spin up calypso
* Load /earn using accounts w/ the following use cases:
* Free plan (clicking the button should take you to the Plans page; learn more should point you to documentation)
* Any paid plan w/o Stripe connection (clicking the button should take you to the Stripe connection funnel)
* Any paid plan w/ Stripe connection (clicking the button should take you to the earnings management screen)
